.env.vercel

```
@@ -2,4 +2,7 @@
 DATABASE_URL=postgresql://neondb_owner:npg_vRJU7LlnYG1y@ep-soft-bush-ab0hbww0-pooler.eu-west-2.aws.neon.tech/neondb?sslmode=require&channel_binding=require
 SESSION_SECRET=Apaddicto2024SecretKey
 NODE_ENV=production
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NODE_ENV key should go before the SESSION_SECRET key

(UnorderedKey)